neard/src/runtime.rs

```diff
@@ -1410,10 +1426,12 @@ impl node_runtime::adapter::ViewRuntimeAdapter for NightshadeRuntime {
             block_timestamp,
             current_protocol_version,
             cache: Some(Arc::new(StoreCompiledContractCache { store: self.tries.get_store() })),
+            #[cfg(feature = "protocol_feature_evm")]
+            evm_chain_id,
         };
         self.trie_viewer.call_function(
             state_update,
-            &view_state,
+            view_state,
```

Comment:
Particular reason for this change?

Reply:
If the target method has to consume it, then it's better to move it. This way if the caller doesn't need the object, then we avoid unnecessary cloning. If the caller still needs the object, then the caller can clone it and we end up with the same number of clones as if passed by the reference.